### DISCUSSION TITLE
Thảo luận tuần 1

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#17] [General]
Bài thực hành word "Trình bày suy nghĩ của bản thân về vạn vật xung quanh"

**Comment by LTGHan** [comment]
Hi

**Comment by HuynhLeBaoTran** [comment]
hii mn

**Comment by GiaHan06** [comment]
hé lô hé lô

**Comment by chaudambaotram** [comment]
hiii

**Comment by GiaHan06** [comment]
cái bài làm của mình là thầy có yêu cầu sd 3 thành ngữ/ tục ngữ/ ca dao/ thơ tự sáng tác á
mấy pà muốn sáng tác hong:3

**Reply by NguyenNgocQuynhAnh** [reply]
hok pé ơi

**Reply by HuynhLeBaoTran** [reply]
hỏng bit sáng tác lun

**Reply by LTGHan** [reply]
Sd cao dao tục ngữ i nha

**Reply by GiaHan06** [reply]
sao hok có icon sad zị

**Reply by LTGHan** [reply]
Sáng tác tới xong c3 ch ra bài nào lun á

**Comment by LTGHan** [comment]
Z h mình tìm ca dao tục ngữ trc hả mn

**Reply by HuynhLeBaoTran** [reply]
p tự làm lun ó

**Comment by GiaHan06** [comment]
ô cê zị h mình tìm đi, mỗi người tìm 2-3 câu gì đó thấy ok gòi gửi lên nha, hoặc 1 cái trang nào đó mà nhiều câu hay qá hong bik chọn cái nào thì gửi link zô luôn:>

**Comment by GiaHan06** [comment]
ê h mình thống nhất 1 giờ để chọn ra 3 câu đi
tui định tối nay mà thấy sáng mai có toán lí hóa :'(

**Reply by GiaHan06** [reply]
sáng thứ 6 gảnh hok

**Reply by chaudambaotram** [reply]
cóa nha

**Reply by HuynhLeBaoTran** [reply]
okeyy

**Reply by GiaHan06** [reply]
@LTGHan @NguyenNgocQuynhAnh @HuynhLeBaoTran thì sao mấy pà

**Reply by LTGHan** [reply]
Ok

**Reply by NguyenNgocQuynhAnh** [reply]
tui oke hết á

**Comment by GiaHan06** [comment]
chốt đơn 9h sáng thứ 6 nha mấy pà

**Comment by chaudambaotram** [comment]
t làm xong rùi nha! để ở trên phần Project nha.Nó cz dài lém í.Có thể sai nữa nên có j mọi người sửa giúp tui nha.Yêu mọi người
